Index buffer size is using vertex_buffer_size

jherico opened the pull request:
## Description

Apparent simple copy paste error.  When creating the index buffer for the GUI, it uses `vertex_buffer_size` instead of `index_buffer_size`

## Checklist:

Ugh, seriously, guys, come on.

jherico committed:
Index buffer size is using vertex_buffer_size

wasimabbas-arm commented:
LGTM, please revert the copyright headers commit.

ZandroFargnoli reviewed (changes requested):
@jherico Thank you for the fixes, I see you have a few PRs changing many of the files. You should only need to change the copyright for files you've added/modified.

If you can revert your commits and just add -2020 to the `.cpp` file we can then merge it.

jherico commented:
@ZandroFargnoli  @wasimabbas-arm copyright headers reverted.  However, the gui file I changed contains two copyright lines, one for "arm and contributors" which I updated, and one specifically for Sascha, which I didn't touch.  I don't know if the CI check is smart enough to differentiate between them.

wasimabbas-arm commented:
> @ZandroFargnoli @wasimabbas-arm copyright headers reverted. However, the gui file I changed contains two copyright lines, one for "arm and contributors" which I updated, and one specifically for Sascha, which I didn't touch. I don't know if the CI check is smart enough to differentiate between them.

This is a good question. We should probably change the copyright script to look for all copyrights. I have created https://github.com/KhronosGroup/Vulkan-Samples/issues/42
